java/ql/src/semmle/code/java/dispatch/DispatchFlow.qll

```diff
@@ -18,6 +18,21 @@ private Callable dispatchCand(Call c) {
   result = viableImpl_inp(c)
 }
 
+/** Holds if `t` is a type that is relevant for dispatch flow. */
+predicate relevant(RefType t) {
```

Comment:
Should the added predicate and class perhaps remain `private` implementation details, or are they intended to be used elsewhere?

Reply:
No, they should indeed be private.  Fixed.